centos_ci.sh

```diff
@@ -292,14 +292,42 @@ function perform_release() {
   curl http://minibot.19cf262c.svc.dockerapp.io:9009/hubot/centosci -H "Content-Type: application/json" -d '{"payload":{"status":"success","message":'"\"$MESSAGE\""',"url":'"\"$URL\""'}}' || true
 }
 
+function perform_docs_publish() {
+  # REPO and BRANCH variables are populated via https://ci.centos.org/job/minishift-docs
+  REPO_OWNER=$(echo $REPO | cut -d"/" -f4)
+  cd gopath/src/github.com/minishift/minishift
+
+  # Get changes from remote repo/branch
+  if [[ "$REPO_OWNER" != "minishift" ]]; then
```

Comment:
So this if statement is for the case that only the branch is specified? You don't need to specify both parameters? That is not reflected in the docs.